codemod/undecorate.spec.ts

```diff
@@ -2,8 +2,15 @@ import { applyTransform } from "jscodeshift/dist/testUtils"
 const dedent = require("dedent-js")
 import * as u from "./undecorate"
 
-function convert(source: string): string {
-    return applyTransform(u, {}, { source: dedent(source), path: "unittest" }, {})
+function convert(
+    source: string,
+    options: {
+        ignoreImports?: boolean
+        keepDecorators?: boolean
```

Comment:
Just a reminder, there should be a separate option for `@observer` unless you changed your mind about it. I still believe people shouldn't be forced to it if they only want to convert decorators in their stores.

Reply:
Let's see if anybody requests it. I'd be surprised if people want to keep decorators just for class components. If they keep them, it'd be for the class fields where they offer way more value.